src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1012,10 +1015,15 @@ internal void OnInsufficientMp()
         var animationPlayer = mutationPointsBar.GetNode<AnimationPlayer>("FlashAnimation");
         animationPlayer.Play("FlashBar");
 
-        GUICommon.Instance.PlayCustomSound(unableToPlaceHexSound);
+        PlayInvalidActionSound();
+    }
+
+    internal void OnActionBlockedWhileMoving()
```

Comment:
I suppose this method could be removed in favor of just calling `PlayInvalidActionSound()` in other places?

Reply:
I floated around the idea of having a "Cancel Move" button, that would flash. That's why this is a separate method. I'll open an issue later about adding that button.

Reply:
Ah alright then.